README.md

```diff
@@ -251,26 +252,29 @@ use_default_rules: true
 verbosity: 1
 ```
 
-
 Pre-commit
-==========
+----------
 
-To use ansible-lint with [pre-commit](http://pre-commit.com/), just 
-add the following to your local repo's `.pre-commit-config.yaml` file. 
-Make sure to change `sha:` to be either a git commit sha or tag of 
+To use ansible-lint with [pre-commit](http://pre-commit.com/), just
+add the following to your local repo's `.pre-commit-config.yaml` file.
+Make sure to change `sha:` to be either a git commit sha or tag of
 ansible-lint containing `hooks.yaml`.
 
 ```yaml
-- repo: https://github.com/willthames/ansible-lint.git
+- repo: https://github.com/ansible/ansible-lint.git
   sha: v3.3.1
   hooks:
     - id: ansible-lint
       files: \.(yaml|yml)$
```

Comment:
This is redundant.

Reply:
Ref: https://github.com/willthames/ansible-review/pull/48#discussion_r225539603

Reply:
@webknjaz As you saying we can assume that user has `ansible-lint` installed and in PATH, rather than having to detail download.

https://github.com/willthames/ansible-review/pull/48/files#diff-04c6e90faac2675aa89e2176d2eec7d8R196 still has this in